scale/storage/serializers.py

```diff
@@ -111,9 +110,30 @@ class ScaleFileBaseSerializer(ModelIdSerializer):
     source_ended = serializers.DateTimeField()
 
     last_modified = serializers.DateTimeField()
+    
+class ScaleFileBaseSerializerV6(ModelIdSerializer):
```

Comment:
For right now, let's just put file_name in the base file serializer and move everything else to the file serializer.